raiwidgets/raiwidgets/model_analysis_dashboard_input.py

```diff
@@ -25,8 +25,8 @@ def __init__(
         self._is_classifier = model is not None\
             and hasattr(model, SKLearn.PREDICT_PROBA) and \
             model.predict_proba is not None
-        self._dataframeColumns = None
         self.dashboard_input = analysis.get_data()
+        self._dataframeColumns = self.dashboard_input.dataset.feature_names
```

Comment:
`_dataframeColumns` should be use snake case and we should avoid naming variables with the type "dataframe" like this. Maybe `_dataset_features`, `_dataset_feature_names`, or `_dataset_columns`?